packaging/nntrainer.spec

```diff
@@ -196,15 +240,32 @@ cp -r result %{buildroot}%{_datadir}/nntrainer/unittest/
 %{_includedir}/nntrainer/tensor.h
 %{_includedir}/nntrainer/lazy_tensor.h
 %{_includedir}/nntrainer/tensor_dim.h
-%{_includedir}/nntrainer/nntrainer.h
 %{_includedir}/nntrainer/nntrainer_log.h
 %{_includedir}/nntrainer/nntrainer_logger.h
 %{_includedir}/nntrainer/optimizer.h
 %{_includedir}/nntrainer/util_func.h
 %{_includedir}/nntrainer/parse_util.h
-%{_libdir}/*.a
 %{_libdir}/pkgconfig/nntrainer.pc
 
+%files devel-static
+%{_libdir}/*.a
+%exclude %{_libdir}/libcapi*.a
+
+%if %{with tizen}
+%files -n capi-nntrainer
+%manifest capi-nntrainer.manifest
+%license LICENSE
+%{_libdir}/libcapi-nntrainer.so
+
+%files -n capi-nntrainer-devel
+%{_includedir}/nntrainer/nntrainer.h
+%{_libdir}/pkgconfig/capi-nntrainer.pc
```

Comment:
Is this okay not to add *.so file here?

Reply:
i requres cpi-nntrainer which has .so. no need to include *.so.

Reply:
Got it. Thanks for the explanation :)